docs/src/clusters/available.md

```diff
@@ -6,8 +6,8 @@ pagination_label: Available Solana Clusters
 
 Solana maintains several different clusters with different purposes.
 
-Before you begin make sure you have first
-[installed the Solana command line tools](../cli/install.md)
+Before you begin, make sure you have first built the Solana command line tools.
+See [Installing the Solana Command Line Tools](../cli/install.md#build-from-source).
```

Comment:
This page is a little trickier:
- There is mention to the CLI (ie `solana config set`) AND
- There is mention to `agave-validator` (ie the example validator scripts).

I think we can tweak the wording to reflect that. Maybe something like "Make sure you have the Solana CLI and `agave-validator` binary available".

Reply:
That being said, the `solana config` commands really aren't that valuable on this page. We have similar content at below page, and we could potentially move the cluster specific commands to that page (in a separate PR to be landed first) such that this page does not need to mention `solana` CLI specifically:
https://docs.anza.xyz/cli/examples/choose-a-cluster

Reply:
Marking this one resolved - let's leave it alone for now